client/components/helpCentre/contactUs/contactUsForm.tsx

```diff
@@ -254,6 +254,7 @@ export const ContactUsForm = (props: ContactUsFormProps) => {
 					us you will need to use the email you registered with.
 				</p>
 				<Input
+					data-qm-masking="blocklist"
```

Comment:
This won't have any effect as it's a React component so the data attribute won't be propagated through to the rendered `input` element that QM will be looking at.

Reply:
After chatting offline I have removed similar instances and put in on the actual HTML element